cmd/gateway/main.go

```diff
@@ -238,8 +238,8 @@ func (flags GatewayFlags) NewGateway(ctx context.Context) (gw minio.Gateway, err
 	return miniogw.NewStorjGateway(
 		project,
 		access,
-		storj.Cipher(flags.Enc.PathType).ToCipherSuite(),
-		flags.GetEncryptionScheme().ToEncryptionParameters(),
+		storj.CipherSuite(flags.Enc.PathType),
+		flags.GetEncryptionScheme(),
```

Comment:
From jira description and from PR description it looks we should `eliminate EncryptionScheme and use EncryptionParameters only everywhere.`. Why we still have this?

Reply:
https://github.com/storj/storj/pull/2363#discussion_r299103370